.github/workflows/py_checks.yml

```diff
@@ -151,3 +151,31 @@ jobs:
       - name: Run Bandit
         run: python -m bandit -r ./ -f screen
         working-directory: src/bindings/python/src/compatibility/openvino
+
+        # Python API 2.0 tests Flake code-style
+      - name: Changed Files
+        uses: tj-actions/changed-files@v36.1.0
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v36
+
+      - name: Run flake8 on modified python tests in layer_tests
+        if: contains(steps.changed-files.outputs.modified_files, 'openvino/tests/layer_tests/')
+        run: |
+          for file in ${{ steps.changed-files.outputs.modified_files }}; do
+              if [[ $file == "openvino/tests/layer_tests/"* ]]; then
+                  python -m flake8 "$file" --config=setup.cfg
+              fi
+          done
+
+      - name: Run mypy on modified python tests in layer_tests
+        if: contains(steps.changed-files.outputs.modified_files, 'openvino/tests/layer_tests/')
+        run: |
+          for file in ${{ steps.changed-files.outputs.modified_files }}; do
+                if [[ $file == "openvino/tests/layer_tests/"* ]]; then
+                    python -m mypy "$file" --config-file ./setup.cfg
```

Comment:
can we create a tax.ini file for all such checks?

Reply:
> Also, I see that other tests in the `py_checks.yml` file are not being checked with `mypy`. I think we can drop it.

Shall I comment it out and leave , so that it might be used in the future ? or remove that code from the file ?@p-wysocki